files/build_templates/physyncd.service.j2

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=physyncd service
+Requires=database.service updategraph.service
+{% if sonic_asic_platform == 'broadcom' %}
+Requires=opennsl-modules.service
+ConditionPathExists=!/usr/share/sonic/hwsku/gearbox_config.json
```

Comment:
on the host, I am not sure this is the correct path, it is likely /usr/share/sonic/devices/onie_platform/hwsku

Reply:
This looks ok to me. In the docker:

root@sonic:/# ls /usr/share/sonic/hwsku/
buffers.json.j2		 gearbox_config.json	port_config.ini
buffers_defaults_def.j2  lanemap.ini		qos.json.j2
buffers_defaults_t0.j2	 pai.profile		sai.profile
buffers_defaults_t1.j2	 pg_profile_lookup.ini	td2-s6000-32x40G.config.bcm
context_config.json	 phy1_config_1.json